src/hooks/datasets.ts

```diff
@@ -65,18 +66,25 @@ export function useMonitorsAndGeostoriesPaginated(
     MonitorsAndGeostoriesPaginatedParsed
   >
 ) {
+  const { theme, ...restParams } = params || { theme: undefined };
+  const themeQuery =
+    theme && theme.length > 0
+      ? `?${theme.map((t) => `theme=${encodeURIComponent(t)}`).join('&')}`
+      : '';
   const fetchMonitorAndGeostories = () =>
     API.request<MonitorsAndGeostoriesPaginated>({
       method: 'GET',
-      url: '/monitors-and-geostories',
-      params: { ...params, pagination: true },
+      url: `/monitors-and-geostories${themeQuery}`,
```

Comment:
Why `themeQuery` is here instead of in the params?

Reply:
because API doesn't accept theme: string[], I have to send theme=string&theme=string